src/main/java/org/alliancegenome/curation_api/jobs/executors/AlleleDiseaseAnnotationExecutor.java

```diff
@@ -47,6 +48,8 @@ public void runLoad(BulkLoadFile bulkLoadFile, Boolean cleanUp) {
 
 			IngestDTO ingestDto = mapper.readValue(new GZIPInputStream(new FileInputStream(bulkLoadFile.getLocalFilePath())), IngestDTO.class);
 			bulkLoadFile.setLinkMLSchemaVersion(getVersionNumber(ingestDto.getLinkMLVersion()));
+			if (StringUtils.isNotBlank(ingestDto.getAllianceMemberReleaseVersion()))
+				bulkLoadFile.setAllianceMemberReleaseVersion(ingestDto.getAllianceMemberReleaseVersion());
```

Comment:
Wondering if we can get rid of this duplication of logic?

Reply:
I can't think of a clean way of doing this, especially as the logic is different for FMS files.  Open to suggestions though.